plugins/rapid7_insightidr/help.md

```diff
@@ -959,39 +959,41 @@ Example output:
 ```
 {
   "log": {
-    "id": "0b9a242d-d2fb-4e42-8656-eb5ff64d652f",
-    "links": [
-      {
-        "href": "https://example.com",
-        "rel": "Related"
-      }
-    ],
-    "logsets_info": [
-      {
-        "id": "bc38a911-65f1-4755-cca3-a330a6336b3a",
-        "links": [
-          {
-            "href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85",
-            "rel": "Self"
-          }
-        ],
-        "name": "Unparsed Data",
-        "rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a"
+    "log": {
```

Comment:
Just wondering why its so nested? `log` in `log`?

Reply:
Are the examples in `plugin.spec.file` look fine?

Reply:
fixed 🙂